### PR TITLE
Feature --- Course Page Redesign

### PR DESCRIPTION
**Pull request by MusahMusah**
Course Page redesign.
 Seamless flow of Carousel Items and Search Bar Added.

image

image

image

### CODE REVIEW

**Comment by tomideo1** [comment]
> Course Page redesign.
> Seamless flow of Carousel Items and Search Bar Added.
> 
> image
> 
> image
> 
> image

The Searchbar should be a white background, kindly change this

**Comment by tomideo1** [comment]
You Have some merge conflicts , kindly rebase with dev branch and resolve this conflicts and re push

File: src/resources/views/learners/base/settings.blade.php [hunk 1]
```diff
@@ -48,8 +74,45 @@
...
+
+            data: {
+                firstName: '',
+                lastName: '',
+                email: '',
+                phone: '',
+                errors: [],
+            },
+            methods: {
+                validateForm: function () {
```
Comment (tomideo1) [inline src/resources/views/learners/base/settings.blade.php:95]: can we use the vue `vuelidate` component here so we have a general validation library across board

Comment (tomideo1) [reply src/resources/views/learners/base/settings.blade.php:95]: if this is not possible, please comment out your reasons in this review

File: src/resources/js/components/CoursesCarousel.js [hunk 2]
```diff
@@ -0,0 +1,367 @@
+Vue.component('courses-carousel', {
+    template: 
+    `
+    <div>
+    <h2 class="pl-4 font-weight-bold" style="margin-top: 2em">Let's start learning, MusahMusah</h2>
+    <div v-for="(item, index) in items">
+        <h3 class="pl-4 p-3 font-weight-bold" :title="item.title">Top courses in <a href="" style="color: blue">{{ item.category }} </a></h3>
+        <div class="row visible-courses">
+            <div class="col-12">
+            <div :id="'myCarousel'+index" class="carousel slide" data-ride="carousel" data-interval="0">
```
Comment (tomideo1) [inline src/resources/js/components/CoursesCarousel.js]: kindly convert this to a re usable component, and lets try and not be hardcoing data, you can always spool a dummy array for this and loop through

File: src/resources/views/tenants/course/show.blade.php [hunk 3]
```diff
@@ -2,11 +2,24 @@
...
+            content: ">>";
+        }
+    </style>
 @endsection
 
 
 @section('body_content_main')
-    <div id="course">
+@include('modules-lms-base::navigation',['type' => 'tenant'])
+<nav>
```
Comment (tomideo1) [inline src/resources/views/tenants/course/show.blade.php:15]: also turn this breadcrumb to a reusable component please

**Comment by MusahMusah** [comment]
Thanks for thr review,  noted and will respond accordingly.

**Pull request closed without merging**